webpages/popup/index.html

```diff
@@ -11,6 +11,7 @@
       <div id="title">
         <img src="../../images/icon.svg" id="logo" alt="Logo" />
         <span id="title-text">Scratch Addons</span>
+        <span><a id="version" href="https://scratchaddons.com/changelog">...</a></span>
```

Comment:
Why specify a `href` if you'll then trap the click event and open a different page?

Reply:
Because I forgot to remove that. 😄